test/models/user_test.rb

```diff
@@ -228,4 +240,40 @@
       locks.each { |lock| lock.reload.deleted_at.wont_be_nil }
     end
   end
+
+  describe "#admin_for_project?" do
+    it "is true for a user that has been granted the role of project admin" do
+      users(:deployer_project_admin).is_admin_for?(projects(:test)).must_equal(true)
+    end
+
+    it "is false for users that have not been granted the role of project admin" do
+      users(:viewer).is_admin_for?(projects(:test)).wont_equal(true)
+      users(:deployer).is_admin_for?(projects(:test)).wont_equal(true)
+      users(:admin).is_admin_for?(projects(:test)).wont_equal(true)
+      users(:super_admin).is_admin_for?(projects(:test)).wont_equal(true)
+    end
+  end
+
+  describe "#deployer_for_project?" do
+    it "is true for a user that has been granted the role of project deployer" do
+      users(:viewer_project_deployer).is_deployer_for?(projects(:test)).must_equal(true)
+    end
+
+    it "is true for a user that has been granted the role of project admin" do
+      users(:deployer_project_admin).is_deployer_for?(projects(:test)).must_equal(true)
+    end
+
+    it "is false for users that have not been granted the roles of project deployer or project admin" do
+      users(:viewer).is_deployer_for?(projects(:test)).wont_equal(true)
+      users(:deployer).is_deployer_for?(projects(:test)).wont_equal(true)
+      users(:admin).is_deployer_for?(projects(:test)).wont_equal(true)
+      users(:super_admin).is_deployer_for?(projects(:test)).wont_equal(true)
```

Comment:
Mmm yeah, now I understand. It is a *little* odd, but is more like a naming problem than behavioral.